### ISSUE TITLE
Feature Request: Easily add and remove current directory

### ISSUE THREAD

**Issue by grafst** [mosen/mysides#30]
Great App!!

It is very cumbersome to do `mysides add $(basename $(pwd)) file:///$(pwd)`
It would be very cool to do: `mysides current add` (and `mysides current remove`

**Comment by andrewcrook** [comment]
Does not look as if its had a release in 7 years 
Could you use a shell alias or script file on path to get around this?

**Comment by mosen** [comment]
I will try to find some time but I don't work in the Mac space anymore. I need to drop a universal2 release anyway even if no code changes

**Comment by acsr** [comment]
> I will try to find some time but I don't work in the Mac space anymore. I need to drop a universal2 release anyway even if no code changes

@mosen are you aware of this fork (no Pull Request):
https://github.com/mpermann/mysides he started to create the Universal build...

**Comment by mosen** [comment]
Nope. Happy to merge it back if you want. Didn't do much mac admin for years but happy to do a universal build

**Comment by acsr** [comment]
Cool! Give it a try
